src/main/java/Duke.java

```diff
@@ -1,10 +1,153 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    private static String logo  = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static ArrayList<Task> list;
+
+    public static void main(String[] args) throws DukeException {
+        list = new ArrayList<>();
+
         System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you today, Master?");
+        Scanner scanner = new Scanner(System.in);
+        String text = scanner.nextLine().toLowerCase();
+        boolean open = true;
+        while (open) {
+            try {
+                // Bye
+                if (text.equals("bye") || text.equals("exit")) {
```

Comment:
In this section, user input is parsed and processed. Would it be better if a Parser class was used to handle this?